src/transformers/trainer.py

```diff
@@ -1354,3 +1422,25 @@ def prediction_step(
         if labels is not None:
             labels = labels.detach()
         return (loss, logits.detach(), labels)
+
+    def floating_point_ops(self, model: nn.Module, inputs: Dict[str, Union[torch.Tensor, Any]]):
+        """
+        For models with a `floating_point_ops` method (e.g. models that inherit from `PretrainedModel`), uses
```

Comment:
```suggestion
        For models that inherit from :class:`~transformers.PretrainedModel`, uses
```
I don't think it's worth mentioning the method since people are supposed to use our models for Trainer, that all inherit from `PretrainedModel`.